server/chat-plugins/scavenger-games.ts

```diff
@@ -213,6 +220,46 @@ const TWISTS: {[k: string]: Twist} = {
 		},
 	},
 
+	spamfilter: {
+		name: 'Spam Filter',
+		id: 'spamfilter',
+
+		desc: 'Every wrong answer adds 30 seconds to your final time!',
+
+		onIncorrectAnswer(player: ScavengerHuntPlayer, value: string) {
+			if (!player.incorrect) player.incorrect = [];
+			const id = `${player.currentQuestion}-${value}`;
+			if (player.incorrect.includes(id)) return;
```

Comment:
This wouldn't be every wrong answer, no? This would be every unique wrong answer.

Reply:
Also, why not use a `Set`?

Reply:
yeah that's correct, no need to beat a dead horse.  Let's not punish people for not using their brains (too much)

I don't code much so I never really looked into sets.  I assume it's kinda the same since Sets just have unique ids? I feel like it's fine the way it is now zzz

Reply:
The advantage to using a `Set` is that you wouldn't have to worry about forgetting the `if (player.incorrect.includes(id)) return;` line. Refactoring to use a `Set` can go in a separate PR, though.